account_analytic_distribution_manual/models/base.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import json
+
+from lxml import etree
+
+from odoo import api, models
+from odoo.tools.misc import frozendict
+
+
+class AnalyticMixin(models.AbstractModel):
+    _inherit = "base"
+
+    @api.model
+    def get_view(self, view_id=None, view_type="form", **options):
```

Comment:
Why not doing this in the mixin? Overriding base model is too much and must be avoided.

Reply:
Because in the mixin, this code is only executed on models that inherit from this mixin. For example, the account.move model does not inherit from this mixin. Therefore, when processing views, it embeds a view into account.move.line, but this code is not executed. For this reason, I implemented the code in the base model. I check if the model has the field manual_distribution_id or if it has a one2many field where the related model has the field.

Reply:
Because in the mixin, this code is only executed on models that inherit from this mixin. For example, the account.move model does not inherit from this mixin. Therefore, when processing views, it embeds a view into account.move.line, but this code is not executed. For this reason, I implemented the code in the base model. I check if the model has the field manual_distribution_id or if it has a one2many field where the related model has the field.

Reply:
Please put all this explanation as comments in the code, and next time, anticipating that the decision is going to be controversial, please document it in advance.

Reply:
done